script/install.sh

```diff
@@ -421,6 +421,17 @@ resolve_latest() {
             logV "Determining latest ldc-beta version ($url)."
             COMPILER="ldc-$(fetch $url)"
             ;;
+        ldc-nightly)
+            local CIurl=https://github.com/ldc-developers/ldc/releases/tag/CI
+            logV "Finding latest ldc-nightly binary package (at $CIurl)."
+            local package
+            package="$(curl -s $CIurl | grep -o "ldc2-[0-9a-f]*-$OS-$ARCH-[0-9]*.tar.xz" | head -n 1)"
```

Comment:
Hopefully we can replace this with a more robust way of getting the latest CI build, but we can live with it for now.

Reply:
The uploads are not sorted and old ones aren't deleted, so that results in a random artifact, not the latest one.
Who's updating the sha of the CI tag btw?

Reply:
(nice, you spotted the problem too)